.github/workflows/ci-pipeline.yml

```diff
@@ -404,8 +404,8 @@ jobs:
             }
 
             case "$rule_name" in
-              "config-reloaders" | "etcd" | "kube-state-metrics" | "kubernetes-apps" | "kubernetes-resources" | "kubernetes-storage" | "prometheus" | "prometheus-operator") kebab_to_camel "$rule_name" ;;
-              "alertmanager.rules" | "general.rules" | "kubelet.rules" | "node.rules" | "kube-apiserver-availability.rules" | "kube-apiserver-burnrate.rules" | "kube-apiserver-histogram.rules" | "kube-apiserver-slos.rules" | "kube-prometheus-general.rules" | "kube-prometheus-node-recording.rules") kebab_to_camel "${rule_name%.rules}" ;;
+              "config-reloaders" | "etcd" | "kube-state-metrics" | "kubernetes-apps" | "kubernetes-resources" | "kubernetes-storage" | "kube-apiserver-slos" | "prometheus" | "prometheus-operator") kebab_to_camel "$rule_name" ;;
```

Comment:
Is this change expected?

Reply:
Yes, that manifest was in the wrong list.